### PR TITLE
Ability to customise the generated camel-k image

### PR DESCRIPTION
**Pull request by kameshsampath**
Resolving Issue #437

### CODE REVIEW

File: contributing.adoc [hunk 1]
```diff
@@ -90,6 +90,12 @@ After a successful build, if you're connected to a Docker daemon, you can build
 make images
 ```
 
+The above command produces a `camel-k` image with name `docker.io/apache/camel-k`.  Sometimes you might need to produce camel-k images that need to be pushed to the custom repository e.g. `docker.io/myrepo/camel-k`, to do that you can pass a parameter `imgDestination` to the make as shown below:
+
+```
+make imgDestination='docker.io/myrepo' images
```
Comment (lburgazzoli) [inline contributing.adoc]: would prefer to have the variable name uppercase

Comment (kameshsampath) [reply contributing.adoc]: but traditionally the make variables are passed as camelCased ones; anyways will have it updated

Comment (kameshsampath) [reply contributing.adoc]: so I leave it as it is?

Comment (lburgazzoli) [reply contributing.adoc]: yes for the moment, can you fix the conflicts ?

Comment (kameshsampath) [reply contributing.adoc]: fixed them. dont know how it seeped in IIRC i merged them ;)

**Pull request merged**